docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1026,6 +1035,15 @@ addresses for businesses and households, but eventually we might want
 to intentionally include duplicates, e.g. if someone operates a
 business out of their home.
 
+To keep things simple, for now when businesses move to a new address,
+it will be a totally new address. No household or business will ever
+move into their old address.
+
+We have also included a special "employer" to indicate individuals who
+are *not* currently employed.  We assume that 58% of the population is
+employed, which leads to a lot of individual switching to being
+unemployed.  We might need to refine this in the future.
```

Comment:
right now i have that 42% of adults are initialized as unemployed, and the remaining 58% have employers whose sizes fit a lognormal distribution. then people switch to a new job selected uniformly at random. people are steadily becoming increasingly unemployed, and employer sizes are steadily becoming uniform

![image](https://user-images.githubusercontent.com/57412727/182939181-bf75292e-1249-4c67-81d9-8aec057a6268.png)